flasc/data_processing/energy_ratio_wd_bias_estimation.py

```diff
@@ -84,15 +88,17 @@ def _load_er_input_for_wd_bias(
         self,
         wd_bias,
     ):
-        """This function initializes an instance of the EnergyRatioInput
+        """Load EnergyRatioInput objects with bias.
+
+        This function initializes an instance of the EnergyRatioInput
         where the dataframe is shifted by wd_bias for each test turbine.
         This facilitates the calculation of the energy ratios under this
         hypothesized wind direction bias. Additionally, the FLORIS predictions
         for the shifted dataset are calculated and the energy ratios for the FLORIS
         predictions are also calculated.
 
         Args:
-            wd_bias ([float]): Hypothesized wind direction bias in degrees.
+            wd_bias (float): Hypothesized wind direction bias in degrees.
```

Comment:
What does "Hypothesized" refer to here?

Reply:
It's like, we're presuming a bias in this instance, I think the hypothesized is coming from the fact that we'll try a number of possible offsets in the outer loops to find which makes most sense.  Each is a hypothesis?  Maybe that language is coming a little high

Reply:
Ah, makes sense. Ok, I'm good with that.